### ISSUE TITLE
Firefox, Chrome, won't connect to Localhost

### ISSUE THREAD

**Issue by Larrie07** [km4ack/73Linux#23]
Have two computers running Ubuntu 22.04.  In one I manually added ham radio programs, the other I used 73-Linux (last week).  In the 73 linux case, I started PAT Menu, started Packet Modems, all working, but Firefox will not connect to 127.0.0.1. Seems like it wants to go to https:// instead of http:// (at least the error I get, cannot connect, possible proxy error, is same).  Verified that https mode is disabled and auto fill for search bar is disabled.  In the second computer with manually added programs, it works fine.  No error.  Did side by side comparison of all Firefox settings and they are the same.  The files /etc/hosts are the same.  Chrome has same issue.  Same version of programs are being used.  I suspect there is a file somewhere that needs a setting changed.  Searched the web but so far no help.  Any ideas of what I am doing wrong.  

Larrie, AF7NU

**Comment by km4ack** [comment]
Sounds like a port misconfiguration. Open Pat Menu and click on "Quick Stats" See what port pat is running on. Verify that the Pat Server is running. Now click on Settings/Config->Current Config Settings and verify the Pat port on this screen matches what you saw in the Quick Stats.

**Comment by Larrie07** [comment]
Thanks for the hint.  I checked and found Pat Server not active.  Looking around some more and I realized PAT did not install, even though I had selected it in 73-Linux and Pat Menu installed.  I closely watched the terminal window and kept seeing the error Curl:  command not found.  I did a search on the internet found out what Curl is and how to install.  Did that, ran 73-linux again and now PAT installed and it works.  Made a quick connection to a local packet node.  A case of me not knowing Linux super well.
